totalRP3/Locales/enUS.lua

```diff
@@ -495,6 +496,15 @@ Class: 50 characters|r]],
 	CO_REGISTER_AUTO_PURGE_TT = "Automatically remove from directory the profiles of characters you haven't crossed for a certain time. You can choose the delay before deletion.\n\n|cff00ff00Note that profiles with a relation toward one of your characters will never be purged.\n\n|cffff9900There is a bug in WoW losing all the saved data when it reaches a certain threshold. We strongly recommend to avoid disabling the purge system.",
 	CO_REGISTER_AUTO_PURGE_0 = "Disable purge",
 	CO_REGISTER_AUTO_PURGE_1 = "After %s |4day:days;",
+	CO_RELATIONS = "Relations",
+	CO_RELATIONS_DESCRIPTION = "Description",
+	CO_RELATIONS_DESCRIPTION_TT = "|cnGREEN_FONT_COLOR:%p|r for your character's name.|n|cnGREEN_FONT_COLOR:%t|r for the other character's name.",
+	CO_RELATIONS_NEW = "Create new relation",
+	CO_RELATIONS_NEW_TITLE = "Enter the name of the new relation",
+	CO_RELATIONS_NEW_ERROR = "You must enter a name for the new relation.",
+	CO_RELATIONS_DELETE_WARNING = "Are you sure you want to delete the relation %s?",
```

Comment:
```suggestion
	CO_RELATIONS_DELETE_WARNING = "Are you sure you want to delete the relation |cnGREEN_FONT_COLOR:%s|r?",
```
Might be a good idea to make the relation name a bit clearer in the warning with a bit of greenification.

Reply:
The string given to format already contains the relationship color.

Reply:
then ignore me for i am dumb